### ISSUE TITLE
[Shell] On windows there is no Titlebar

### ISSUE THREAD

**Issue by PedroCigaw** [dotnet/maui#4267]
### Description

Using Shell targeting Window has no title bar, which means you can not move the window, minimize, maximize etc

Example here [https://github.com/PedroCigaw/Maui12Shelltitlebar](https://github.com/PedroCigaw/Maui12Shelltitlebar)

### Steps to Reproduce

1. Create Maui App
2. Create a Shell as the main page
3. Run.

### Version with bug

Preview 12 (current)

### Last version that worked well

Preview 11

### Affected platforms

Windows

### Affected platform versions

Window SDK 10.0.19041.0

### Did you find any workaround?

No but some suggestions would be good.

**Comment by Orgbrat** [comment]
Having same issue after upgrading to Maui Preview 12. All window chrome (Title Bar) is missing, which means you can not move the window, minimize, maximize etc...

**Comment by emceelovin** [comment]
Having the same issue. You can move it, you just have to find the right spot on the right of where the titlebar would be. Just west of the minimize button

**Comment by Gogzs** [comment]
Noticed the same issue, think I tracked it down to this PR:
https://github.com/dotnet/maui/pull/3693/files
It happens when you don't set the AppTitleBar.

In NavigationRootManager.cs
```
                        if(_navigationRootView.AppTitleBar != null)
			{
				var nativeWindow = _mauiContext.GetNativeWindow();
				nativeWindow.ExtendsContentIntoTitleBar = true;
				UpdateAppTitleBar(true);
			}
```

If you set the BackgroundColor of your shell to Transparent, you'll see that the chrome buttons are still there, it's just that the shell extended over it.

I'm not sure how to set the AppTitleBar tho, I tried overriding the CreateWindow method in App.Xaml.cs, creating, setting the title and returning that window, but still no title and the chrome still being overlaid by the shell content.
